build/dockerfiles/KServe/entrypoint.sh

```diff
@@ -15,9 +17,18 @@ fi
 REPO_NAME="${1#kit://}"
 OUTPUT_DIR="$2"
 
+if [ -n "$AWS_ROLE_ARN" ]; then
+  AWS_ACCOUNT_ID=$(echo "$AWS_ROLE_ARN" | cut -d: -f5)
+  echo "Logging into AWS ECR $AWS_ACCOUNT_ID.dkr.ecr.$AWS_ECR_REGION.amazonaws.com"
+  aws ecr get-login-password --region $AWS_ECR_REGION | kit login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.$AWS_ECR_REGION.amazonaws.com
+fi
```

Comment:
This will only work for the standard AWS endpoints (i.e. no FIPS, or China-based endpoints), but I can't think of a good way to support those anyways. Is there a clean way to document this in the readme?